pallets/weights/src/pallet_pips.rs

```diff
@@ -0,0 +1,130 @@
+            .saturating_add(DbWeight::get().reads(13 as Weight))
+            .saturating_add(DbWeight::get().writes(9 as Weight))
+    }
+    fn propose_from_committee(c: u32) -> Weight {
+        (122_255_000 as Weight)
+            .saturating_add((4_000 as Weight).saturating_mul(c as Weight))
+            .saturating_add(DbWeight::get().reads(8 as Weight))
+            .saturating_add(DbWeight::get().writes(5 as Weight))
+    }
+    fn amend_proposal(c: u32) -> Weight {
+        (29_957_000 as Weight)
+            .saturating_add((10_000 as Weight).saturating_mul(c as Weight))
+            .saturating_add(DbWeight::get().reads(3 as Weight))
+            .saturating_add(DbWeight::get().writes(1 as Weight))
+    }
+    fn cancel_proposal(c: u32) -> Weight {
+        (87_500_000 as Weight)
+            .saturating_add((17_000 as Weight).saturating_mul(c as Weight))
+            .saturating_add(DbWeight::get().reads(8 as Weight))
+            .saturating_add(DbWeight::get().writes(5 as Weight))
+    }
+    // WARNING! Some components were not used: ["a"]
+    fn vote(c: u32, n: u32) -> Weight {
+        (164_878_000 as Weight)
+            .saturating_add((6_000 as Weight).saturating_mul(c as Weight))
+            .saturating_add((10_471_000 as Weight).saturating_mul(n as Weight))
+            .saturating_add(DbWeight::get().reads(5 as Weight))
+            .saturating_add(DbWeight::get().writes(3 as Weight))
+    }
+    fn approve_committee_proposal(c: u32) -> Weight {
+        (104_857_000 as Weight)
+            .saturating_add((10_000 as Weight).saturating_mul(c as Weight))
+            .saturating_add(DbWeight::get().reads(3 as Weight))
+            .saturating_add(DbWeight::get().writes(3 as Weight))
+    }
+    fn reject_proposal(c: u32) -> Weight {
+        (448_343_000 as Weight)
+            .saturating_add((8_000 as Weight).saturating_mul(c as Weight))
+            .saturating_add(DbWeight::get().reads(8 as Weight))
+            .saturating_add(DbWeight::get().writes(5 as Weight))
+    }
+    fn prune_proposal(c: u32) -> Weight {
+        (112_592_000 as Weight)
+            .saturating_add((7_000 as Weight).saturating_mul(c as Weight))
+            .saturating_add(DbWeight::get().reads(2 as Weight))
+            .saturating_add(DbWeight::get().writes(5 as Weight))
+    }
+    fn reschedule_execution(c: u32) -> Weight {
+        (54_960_000 as Weight)
+            .saturating_add((6_000 as Weight).saturating_mul(c as Weight))
+            .saturating_add(DbWeight::get().reads(6 as Weight))
+            .saturating_add(DbWeight::get().writes(3 as Weight))
+    }
+    // WARNING! Some components were not used: ["c"]
+    fn clear_snapshot() -> Weight {
+        (103_047_000 as Weight)
+            .saturating_add(DbWeight::get().reads(3 as Weight))
+            .saturating_add(DbWeight::get().writes(2 as Weight))
+    }
+    // WARNING! Some components were not used: ["h"]
```

Comment:
Let's try and see.

Reply:
Actually I think `h` is disregarded because it correlates with `b`. We need to rethink how this bench is set up without providing setup parameters to the weight function.